src/Umbraco.Cms.Integrations.Search.Algolia/Models/LibraryMedia.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Text.Json.Serialization;
```

Comment:
This seems to be duplicated in the library project.

Reply:
I see you've removed that now so it's no longer duplicated, but is `LibraryMedia` now the right name?